client/src/App.js

```diff
@@ -42,23 +46,31 @@ function App() {
     <MuiThemeProvider theme={theme}>
       <CssBaseline />
       <BrowserRouter>
-        <Navbar />
-        {loading ? (
-          <Loader />
-        ) : (
-          <Switch>
-            <Redirect exact from="/" to="/login" />
-            <Route path="/explore" exact component={Explore} />
-            <Route path="/login" exact component={Login} />
-            <Route path="/signup" exact component={SignUp} />
-            <Route path="/users/:userId" component={Profile} />
-            <ProtectedRoutes path="/launch" exact component={Launch} />
-            <Route path="*" component={() => 'Page not found'} />
-          </Switch>
-        )}
+        <Elements stripe={stripePromise}>
```

Comment:
We probably want this to be only in the component that we are using it for (future reference)

Reply:
In Strip documentation they suggest it to use it at the root of the application to detect fraudulent attempts anywhere in the whole application. So thats why i used it here.